.changeset/calm-lizards-think.md

```diff
@@ -0,0 +1,93 @@
+'@commercetools-uikit/date-range-field': major
+'@commercetools-uikit/date-time-field': major
+'@commercetools-uikit/localized-multiline-text-field': major
+'@commercetools-uikit/localized-text-field': major
+'@commercetools-uikit/money-field': major
+'@commercetools-uikit/multiline-text-field': major
+'@commercetools-uikit/number-field': major
+'@commercetools-uikit/password-field': major
+'@commercetools-uikit/radio-field': major
+'@commercetools-uikit/search-select-field': major
+'@commercetools-uikit/select-field': major
+'@commercetools-uikit/text-field': major
+'@commercetools-uikit/time-field': major
+'@commercetools-uikit/grid': major
+'@commercetools-uikit/icons': major
+'@commercetools-uikit/async-creatable-select-input': major
+'@commercetools-uikit/async-select-input': major
+'@commercetools-uikit/checkbox-input': major
+'@commercetools-uikit/creatable-select-input': major
+'@commercetools-uikit/date-input': major
+'@commercetools-uikit/date-range-input': major
+'@commercetools-uikit/date-time-input': major
+'@commercetools-uikit/input-utils': major
+'@commercetools-uikit/localized-money-input': major
+'@commercetools-uikit/localized-multiline-text-input': major
+'@commercetools-uikit/localized-rich-text-input': major
+'@commercetools-uikit/localized-text-input': major
+'@commercetools-uikit/money-input': major
+'@commercetools-uikit/multiline-text-input': major
+'@commercetools-uikit/number-input': major
+'@commercetools-uikit/password-input': major
+'@commercetools-uikit/radio-input': major
+'@commercetools-uikit/rich-text-input': major
+'@commercetools-uikit/rich-text-utils': major
+'@commercetools-uikit/search-select-input': major
+'@commercetools-uikit/select-input': major
+'@commercetools-uikit/select-utils': major
+'@commercetools-uikit/text-input': major
+'@commercetools-uikit/time-input': major
+'@commercetools-uikit/toggle-input': major
+'@commercetools-uikit/label': major
+'@commercetools-uikit/link': major
+'@commercetools-uikit/loading-spinner': major
+'@commercetools-uikit/messages': major
+'@commercetools-uikit/notifications': major
+'@commercetools-uikit/pagination': major
+'@commercetools-uikit/primary-action-dropdown': major
+'@commercetools-uikit/spacings-inline': major
+'@commercetools-uikit/spacings-inset': major
+'@commercetools-uikit/spacings-inset-squish': major
+'@commercetools-uikit/spacings-stack': major
+'@commercetools-uikit/stamp': major
+'@commercetools-uikit/tag': major
+'@commercetools-uikit/text': major
+'@commercetools-uikit/tooltip': major
+'@commercetools-uikit/hooks': major
+'@commercetools-uikit/i18n': major
+'@commercetools-uikit/localized-utils': major
+'@commercetools-uikit/utils': major
+'@commercetools-uikit/buttons': major
+'@commercetools-uikit/fields': major
+'@commercetools-uikit/inputs': major
+'@commercetools-uikit/spacings': major
+'@commercetools-frontend/ui-kit': major
+---
+
+bump `@commercetools-uikit/*` and `@commercetools-frontend/ui-kit` to v12
```

Comment:
Nit: ideally we should put the link to the release notes here.

Reply:
dcffe2f